Testbed/Preparing_preconfigured_setup.md

```diff
@@ -0,0 +1,600 @@
+# Preparing connector A and B
```

Comment:
We should reference [this](../Testbed/Preparing_preconfigured_setup.md) from the README.md to guide the user through the Repository.

Reply:
Created  a new issue for this #18